WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php

```diff
@@ -64,13 +64,14 @@ class PrefixAllGlobalsSniff extends AbstractFunctionParameterSniff {
 	public $prefixes = '';
 
 	/**
-	 * Prefix blacklist.
+	 * Prefix blocklist.
```

Comment:
How come this is not `disallowed_list` (or `disallowList`)? Seeing how other replacements are _allowed_

Reply:
I kind of felt this needed a stronger word. "Disallow", for me, has a connotation along the lines of "we choose to disallow", while "block" feels more restrictive/a stronger choice of words, which felt appropriate as these are prefixes which REALLY should be blocked (disallowed) as they WILL cause problems when used.